src/bp/ui-admin/src/Pages/MyAccount/ChangeLanguage.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Classes, Dialog, FormGroup, Intent } from '@blueprintjs/core'
+import { lang } from 'botpress/shared'
+import React, { FC, useState } from 'react'
+import Select from 'react-select'
+import { toastFailure, toastSuccess } from '~/utils/toaster'
+
+interface Props {
+  isOpen: boolean
+  toggle: () => void
+}
+
+interface SelectItem {
+  value: string
+  label: string
+}
+
+const ChangeLanguage: FC<Props> = props => {
+  const languages: SelectItem[] = [
+    { value: 'en', label: 'English' },
+    { value: 'fr', label: 'Français' }
+  ]
+
+  const [language, setLanguage] = useState<SelectItem>(getCurrentLanguage())
+
+  const submit = async event => {
+    event.preventDefault()
```

Comment:
When using BaseDialog, you can set a property onSubmit which handles automatically the preventDefault (and you don'T need to create a form manually)